src/main/java/edu/kis/powp/jobs2d/drivers/DriverManager.java

```diff
@@ -9,19 +9,19 @@
  */
 public class DriverManager {
 
-	private Job2dDriver currentDriver = new LoggerDriver();
+    private Job2dDriver currentDriver = new LoggerDriver();
 
-	/**
-	 * @param driver Set the driver as current.
-	 */
-	public synchronized void setCurrentDriver(Job2dDriver driver) {
-		currentDriver = driver;
-	}
+    /**
+     * @param driver Set the driver as current.
+     */
+    public synchronized void setCurrentDriver(Job2dDriver driver) {
+        currentDriver = driver;
+    }
```

Comment:
Original code used tabs instead of spaces. It is important to keep the original convention. It makes tracking actual changes more difficult.